lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -123,7 +123,7 @@ def empty_if_branch?(node)
         end
 
         def empty_else_branch?(node)
-          node.else_branch.if_type? && !node.else_branch.body
+          node.else_branch&.if_type? && !node.else_branch&.body
```

Comment:
Can you update to the following?

```suggestion
          return false unless (else_branch = node.else_branch)
          
          else_branch.if_type? && !else_branch.body
```